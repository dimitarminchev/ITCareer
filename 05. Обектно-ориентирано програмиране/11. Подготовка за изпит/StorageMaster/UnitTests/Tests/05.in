AddProduct HardDrive -20
RegisterStorage InvalidStorage BadHackerStorage
RegisterStorage Warehouse NiceHackerStorage
SelectVehicle NiceHackerStorage 0
LoadVehicle HardDrive
SendVehicleTo BadHackerStorage 0 NiceHackerStorage
SendVehicleTo NiceHackerStorage 0 BadHackerStorage
END
