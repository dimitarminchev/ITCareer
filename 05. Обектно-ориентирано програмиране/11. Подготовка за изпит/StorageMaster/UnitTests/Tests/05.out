Error: Price cannot be negative!
Error: Invalid storage type!
Registered NiceHackerStorage
Selected Semi
Error: HardDrive is out of stock!
Error: Invalid source storage!
Error: Invalid destination storage!
NiceHackerStorage:
Storage worth: $0.00
